pymemcache/client/base.py

```diff
@@ -813,19 +831,23 @@ def _store_cmd(self, name, values, expire, noreply, cas=None):
             if self.serializer:
                 data, flags = self.serializer(key, data)
             else:
-                flags = 0
+                # flag is passed only by remcached operations
+                if flag:
```

Comment:
If we adopt `flags`, then bit of code can just use the `flags` parameter directly.